torch/testing/_comparison.py

```diff
@@ -283,13 +283,11 @@ def __init__(
         expected: Any,
         *,
         id: Tuple[Any, ...] = (),
-        msg: Optional[str] = None,
```

Comment:
This removes the `msg` parameter from the pairs. That means, the pair has no knowledge of any alternative messages and should always return an `ErrorMeta` with the automatically generated message. The alternative message handling happens in `assert_equal`.